src/main/webapp/app/core/theme/theme.service.ts

```diff
@@ -235,4 +244,33 @@ export class ThemeService {
             this.preferenceSubject.next(theme);
         }
     }
+
+    /**
+     * Hides the notification sidebar as there will be an overlay ove the whole page
+     * that covers details of the exam summary (=> exam summary cannot be read).
+     * @private
```

Comment:
```suggestion
```
Also please explain why this returns a string and what content this string contains.